src/character.cpp

```diff
@@ -3355,6 +3356,16 @@ units::mass Character::bodyweight() const
     return units::from_kilogram( get_bmi() * pow( height() / 100.0f, 2 ) );
 }
 
+units::mass Character::bionics_weight() const
+{
+    units::mass bio_weight = units::from_gram( 0 );
+    for( const auto bio : *my_bionics ) {
+        const item cbm( bio.id.c_str() );
+        bio_weight += cbm.weight();
```

Comment:
```suggestion
        bio_weight += item::find_type( bio.id.c_str() )->weight;
```
No need to create an actual item instance.